gardenstyle.css

```diff
@@ -0,0 +1,265 @@
+
+*{
+    margin: 0;
+    padding: 0;
+    /* box-sizing allows us to include padding and border in element's total width and height  */
+    box-sizing: border-box;
+    text-decoration: none;
+}
+body{
+    background-color: #F7F6F4;
+    font-family: 'Manrope', sans-serif;
+}
+.navbar{
+    position: fixed;
+    width: 100%;
+    padding: 30px;
+    z-index: 999;
+    transition: all 0.3s ease;
+}
+.box{
+    width:40px;
+    height: 35px;
+    background-color: #709D61;
+    float: left;
+    border-radius: 5px;
+    margin-left: 15px;
+}
+.navbar .logo a {
+    font-size: 35px;
+    font-weight: 600;
+    color: #000000;
+}
+.max-width{
+    max-width: 1200px;
+    margin: auto;
+}
+.navbar .max-width{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.navbar.sticky{
+    padding: 15px 0;
+    background: #607d3b;
+}
+.navbar .menu li{
+    list-style: none;
+    display: inline-block;
+}
+.navbar .menu li a{
+    color: #000000;
+    font-size: 16px;
+    font-weight: 500;
+    margin-left: 25px;
+    transition: color 0.3s ease;
+}
+.navbar .menu li a:hover{
+    color: #709D61;
+}
+.menu-btn{
+    color: #000000;
+    font-size: 23px;
+    cursor: pointer;
+    display: none;
+}
+
+
+
+.home{
+    display: flex;
+    height: 100vh;
```

Comment:
Assigning height to dynamic content is not recommended. Please state the purpose

Reply:
Correction done:
.home{
    display: flex;
    height: 100%;
    min-height: 400px;
}